app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,41 @@
+<section class="d-flex-col user-container center">
+  <h2>Sign up</h2>
+
+  <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+    <%= render "devise/shared/error_messages", resource: resource %>
+
+    <div class="field">
+      <%= f.label :name %><br />
+      <%= f.text_field :name, autofocus: true %>
+    </div>
+
+    <div class="field">
+      <%= f.label :bio %><br />
+      <%= f.text_area :bio, autofocus: true %>
+    </div>
+
```

Comment:
- [ ] It would be better if we add the `photo` column field to our form in order to match the wireframe as seen in the [requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md)

Reply:
Done